container/Containerfile

```diff
@@ -13,9 +13,10 @@ RUN /venv/bin/pip install ovirt-imageio.tar.gz
 # Final stage
 
 FROM alpine
-RUN apk add python3
+RUN apk add python3 qemu-img
 RUN mkdir /etc/ovirt-imageio
 COPY --from=build /venv /venv
 COPY conf.d /etc/ovirt-imageio/conf.d
+COPY bin /bin
```

Comment:
I used bin to have the script available in the $PATH and invoke it without the full path.

In other project varies, many of them put it in the root folder, some in any other subfolder. But also the `bin` folder is used, e.g., https://docs.docker.com/develop/develop-images/dockerfile_best-practices/ (multi-stage builds section).

It is just a matter of being able to find the script later in the `ENTRYPOINT` line.

I don't have a preference nor think it makes a lot of difference. If you prefer `app`, is fine by me.

Reply:
Putting the entrypoint script in the path is not important, and it is better to run
it with absolute path anyway, so using bin/ is not required.

If we have multiple files, it makes sense to put all the files in the same directory
instead of adding random files at /. Since /app is documented, it will probably be
familiar to other developers, so it sounds like a good way.

I would copy the entrypoint script to /app, and also store the ticket int the same
place.